src/index.tsx

```diff
@@ -727,6 +730,14 @@ export class Resizable extends React.PureComponent<ResizableProps, State> {
     if (!this.state.isResizing || !this.resizable || !this.window) {
       return;
     }
+    if (event instanceof TouchEvent) {
+      try {
+        event.preventDefault();
```

Comment:
@Voakie If `{ passive: true }`, would this line be ignored?

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

Reply:
Yes, it wouldn't have any effect, since a passive event listener can't call `preventDefault`.

Reply:
@Voakie Thanks for your reply.

I can not understand yet. 

>  it wouldn't have any effect, 

why do you call `preventDefault` ?

Reply:
@bokuweb As you can see on line 465 the `touchmove` event listener is added with `{ passive: false }`. So `preventDefault` will have an effect on the `touchmove` event but not on the `mousemove` event.

Reply:
Oops... sorry, I was mistaken....